### PR TITLE
Shorten nice names by trimming schema name

### PR DESCRIPTION
**Pull request by JGamache-autodesk**
Some attribute names use namespaces that are copies of the schema name. When displaying UI names we want to strip that redundant information.

### CODE REVIEW

**Comment by JGamache-autodesk** [comment]
Before and after for an exported spotlight:
UiNameSpotLight

File: lib/mayaUsd/resources/ae/usdschemabase/ae_template.py [hunk 1]
```diff
@@ -790,9 +788,17 @@ def createAppliedSchemasSection(self):
         # group all instances of a MultipleApply schema together
         # so we can later display them into the same UI section.
         #
-        # By example, if UsdCollectionAPI is applied twice, UsdPrim.GetAppliedSchemas()
-        # will return ["CollectionAPI:instance1","CollectionAPI:instance2"] but we want to group
-        # both instance inside a "CollectionAPI" section.
+        # We do not group the collections API. We rather create a section that
+        # includes the instance in the title. This allows up to optimally trim
```
Comment (feldstj) [inline lib/mayaUsd/resources/ae/usdschemabase/ae_template.py:792]: allows up -> allows us

File: lib/mayaUsd/ufe/UsdAttributeHolder.cpp [hunk 2]
```diff
@@ -206,16 +206,187 @@ Ufe::Value UsdAttributeHolder::getMetadata(const std::string& key) const
...
+                    std::string prefix = names.front();
+                    size_t      dunderPos = prefix.find("__");
+                    if (dunderPos == std::string::npos) {
+                        continue;
+                    }
+
+                    prefix = prefix.substr(0, dunderPos) + typeNameAndInstance.second.GetString()
+                        + ":";
+
+                    // If the parameter name does not start with the templte, it does not belong to
```
Comment (feldstj) [inline lib/mayaUsd/ufe/UsdAttributeHolder.cpp:293]: templte -> template

**Review by feldstj — CHANGES_REQUESTED** [review]

**Comment by feldstj** [comment]
Pretty cool change. Just a few minor spelling issues.

**Review by feldstj — APPROVED** [review]

**Comment by JGamache-autodesk** [comment]
@seando-adsk Ready for merge.

**Review by seando-adsk — APPROVED** [review]

**Pull request merged**